x-pack/plugins/cases/public/components/all_cases/utils/serialize_url_params.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function serializeUrlParams(urlParams: {
+  [key in string]: string[] | string | undefined;
+}) {
+  const urlSearchParams = new URLSearchParams();
+  for (const [key, value] of Object.entries(urlParams)) {
+    if (!value) {
+      break;
```

Comment:
Should we continue with the rest of the values?

Reply:
oopsie